src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.cpp

```diff
@@ -1049,7 +1017,10 @@ SolidMechanicsLagrangianFEM::
                                          parallelDevicePolicy< 32 > >( targetSet,
                                                                        time_n + dt,
                                                                        targetGroup,
-                                                                       keys::TotalDisplacement, // TODO fix use of
+                                                                       extrinsicMeshData::solidMechanics::totalDisplacement::key(), // TODO
+                                                                                                                                    // fix
+                                                                                                                                    // use
+                                                                                                                                    // of
```

Comment:
Hmm, can you do something about the indentation here?

Reply:
I fixed this indentation